NDTensors/ext/NDTensorsCUDAExt/imports.jl

```diff
@@ -4,3 +4,4 @@ import NDTensors:
 import NDTensors.SetParameters: nparameters, get_parameter, set_parameter, default_parameter
 
 import .CUDA: CuArrayAdaptor
```

Comment:
Is this needed?

Reply:
No its not, they changed the name of this variable in `CUDA.jl` I commented it out in my `UnallocatedArray` PR but I can remove the variable in this one if you'd like

Reply:
Sure, may as well remove it here.

Reply:
Okay I removed it!